examples/SFRRT/example_lazy_ellipsoid_2.jl

```diff
@@ -0,0 +1,278 @@
+using Dionysos
+UT = Dionysos.Utils
+SY = Dionysos.System
+SC = Dionysos.Symbolic
+PR = Dionysos.Problem
+
+using Plots, Colors, LinearAlgebra
+using Symbolics
+using IntervalArithmetic
+using LinearAlgebra
+using Mosek
+using MosekTools
+using JuMP
+import Random
+Random.seed!(0)
+
+#######################################################
+################## Problem setting ####################
+#######################################################
+
+# convert square box into intersection of degenerated ellipsoids
+# square box [-x,x]
+function convert_H_into_E(Ub::IntervalBox)
+    nu = length(Ub)
+    Uaux = diagm(1:nu)
+    U = [(Uaux.==i)./Ub[i].hi for i in 1:nu];
+    return U
+end
+
+function unstableSimple()
```

Comment:
yes more or less, this allows the function to be used for different systems by simply passing different symbolic variables as arguments. Otherwise, the different systems will share the same variables.